achitectural-principles/src/main/java/com/epam/mentoring/kiss/Main.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.mentoring.kiss;
+
+public class Main {
+
+    public static void main(String[] args) {
+        if (args.length >= 3) {
```

Comment:
What if `args.length < 3`? I'd rather throw `IllegalArgumentException` explicitly, if we are not going to accept this case. Also, take a look at `guard statement` best practice: https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html